pylint/pyreverse/diagrams.py

```diff
@@ -178,24 +178,41 @@ def extract_relationships(self):
                 except KeyError:
                     continue
             # implements link
-            for impl_node in node.implements:
-                try:
-                    impl_obj = self.object_from_node(impl_node)
-                    self.add_relationship(obj, impl_obj, "implements")
-                except KeyError:
-                    continue
+            if hasattr(node, "implements"):
```

Comment:
Is there a reason why we need to explicitly check for the existence of the `implements` attribute here, when it was not necessary before? `node` should always be of type `ClassDef`, and the `Linker` adds the `implements` attribute when visiting the `ClassDef` nodes.